src/ServiceControl.Transports.SQS/SQSTransportCustomization.cs

```diff
@@ -43,7 +47,9 @@ public override void CustomizeForAuditIngestion(RawEndpointConfiguration endpoin
 
         public override void CustomizeForMonitoringIngestion(EndpointConfiguration endpointConfiguration, TransportSettings transportSettings)
         {
-            CustomizeEndpoint(endpointConfiguration, transportSettings);
+            var transport = endpointConfiguration.UseTransport<SqsTransport>();
+            ConfigureTransport(transport, transportSettings);
+            ConfigurePubSub(transport, transportSettings);
```

Comment:
I was about to make the comment:

Why does Monitoring required pubsub until I realized this was already configured before your changes. 

Now that you split it up should we double check if we actually need it and if not disable pubsub? Or is that outside the scope of this PR?

Reply:
@danielmarbach good idea. I am 99.9% sure it does not require but... :) I'll verify